style.css

```diff
@@ -0,0 +1,12 @@
+@media only screen and (min-width: 992px) {
+  #navbar {
+    display: flex;
+    justify-content: flex-end;
+  }
+}
+
+body {
+  background-color: royalblue;
+  color: white;
+  text-align: center;
```

Comment:
Using text-align to center the content will only center some HTML elements like texts, etc. It a good if I center the content using flexbox so that the whole content can be centered and it is also easy to make them responsive.